build/pipeline-releasebuild-netonly.yaml

```diff
@@ -0,0 +1,122 @@
+
+# Release build for -netonly package should be triggered after the normal release build
```

Comment:
Instead of creating another new build pipeline, can `NET_ONLY_INTERNAL_PACKAGE` be passed as a build variable, and we use the existing logic? the csproj changes can remain as is. 

When `NET_ONLY_INTERNAL_PACKAGE` is passed as a build variable then we should not build the client package, we should only build the -netonly_internal package

Reply:
I tried something like this and our tasks usually build the whole solution. So if we exclude most of the targets, the test projects and devapps won't build. We then would have to add more logic to probably exclude certain tasks, and I thought that would complicate the current pipeline. And even if we build only the main proj; we'd still have to add logic to do that. If there's a way to just add minimal flags to the current pipeline, then maybe that'll work, but I'm not sure.